Collapsible markdown (#20)
cybertec-postgresql/today-i-learned-content: posts/TEMPLATE/index.md

---
date: 2030-12-31 # Auto updates on publish.

> title: Collapsible markdown

Reviewer: ```suggestion
title: Collapsible Markdown
```

author: kieran.kaelin@cybertec.at
tags: ["markdown"] # max. 10 tags; lowercase; dash-separated

> description: "A short example on how to create collapsible markdown" # max. 300 chars.

Reviewer: ```suggestion
description: "A short example on how to create collapsible Markdown" # max. 300 chars.
```

---

> When presenting code or logs in your markdown, things tend to get out of hand quickly. \

Reviewer: ```suggestion
When presenting code or logs in Markdown, things tend to get out of hand quickly. \
```

> To mitigate this, you can use collapsible markdown to hide certain parts of your document.

Reviewer: ```suggestion
The `<details>` and `<summary>` HTML tags can be used to mitigate this, which hide certain parts of your document. Be aware, that Markdown specific syntax constructs within those HTML tags are only guaranteed to be rendered correctly by [CommonMark](https://spec.commonmark.org/0.29/#html-block) and / or [GFM](https://github.github.com/gfm/) compliant parsers (for example the GitHub Markdown parser).  
```

```md
<details>
	<summary>Click to expand this section!</summary>
	<h5>A nice Javascript pitfall!</h5>

	```javascript
	console.log(['1', '7', '11'].map(parseInt));
	```
</details>
```

> This markdown creates the following result:

Reviewer: ```suggestion
This Markdown snippet creates the following result:
```
